lib/Listener/BeforeTemplateRenderedListener.php

```diff
@@ -24,7 +25,7 @@ public function handle(Event $event): void {
 			return;
 		}
 
-		if ($this->capabilitiesService->hasWASMSupport()) {
+		if ($this->capabilitiesService->hasWASMSupport() && $event->getResponse()->getApp() === Application::APPNAME) {
```

Comment:
As far as I remember this will not work as we require the headers to be set e.g. on the files app as it is the one that iframes Collabora directly. 

@caolanm Since you worked on that, can you provide some steps how to test and verify that the wasm support still fully works with that change?

Reply:
If not, would it be acceptable to hardcode `'files'` (and probably `'files_sharing'`) as well here?

Reply:
Depends, in theory collabora could be loaded by any app, e.g. talk when using the viewer to show a file shared to a conversation.

Reply:
Though it is arguable what benefit the wasm experimental feature has there